docs/dev/checks/README.md

```diff
@@ -133,3 +125,91 @@ checks code.
 [collector]: /pkg/collector
 [datadog_checks_base]: https://datadog-checks-base.readthedocs.io/en/latest/
 [developer_docs]: https://docs.datadoghq.com/developers/
+
+## Running checks with a local Agent Build
+### Custom Checks
+Scenario: You have implemented a custom check called `hello_world` and you'd like
+to run this with a local agent build.
+
+1. Place the configuration file `hello_world.yaml` in `dev/dist/conf.d/` folder
+2. Place the pthon code in `dev/dist/` folder
+2. Run `inv agent.build` as usual. This step will copy the contents
+   of `dev/dist` into `bin/agent/dist`, which is where the agent will be
+   able to find your code.
+
+The resulting directory structure should look like:
+```
+dev/dist
+├── conf.d
+│   └── hello_world.yaml
+├── hello_world.py
+└── README.md
+```
+
+### Standard Checks
+There are a number of checks that ship with a full build of the Agent, but when
+you're developing against a local build, you will not get any of these python
+checks out of the box, you must install them.
+
+> Note the following instructions install python packages to the python user
+> install directory. This could conflict with existing packages.
+
+The list of checks currently shipping with the agent lives in the
+[integrations-core repo
+here](https://github.com/DataDog/integrations-core/blob/master/requirements-agent-release.txt)
+
+Say you want to test the `openmetrics` check.
+
+1. Clone `integrations-core` locally. (Optionally checkout the git tag
+   corresponding to the version you want to test)
+2. From integration-core directory, install `datadog-checks-base` as a pre-req.
+   `pip3.10 install --user './datadog_checks_base[deps]'`
```

Comment:
Generally better for readability/copy-paste to put these in fenced codeblocks, but I can't do that in code-suggestions cos it breaks the suggestion 😅